libs/sdk-core/src/breez_services.rs

```diff
@@ -905,6 +909,38 @@ impl BreezServices {
         Ok(())
     }
 
+    fn persist_new_bolt11_payment_attempt(
+        &self,
+        invoice: &LNInvoice,
+        amount_msat: u64,
+    ) -> Result<(), SendPaymentError> {
+        self.persister.insert_or_update_payments(&[Payment {
+            id: invoice.payment_hash.clone(),
+            payment_type: PaymentType::Sent,
+            payment_time: SystemTime::now().duration_since(UNIX_EPOCH)?.as_secs() as i64,
+            amount_msat,
```

Comment:
This can be set to 0 and instead call `persister.insert_payment_external_info()` to store the attempted_amount_msat in the payment_external_info sync table.
```suggestion
            amount_msat: 0,
```

Reply:
> This can be set to 0 and instead call `persister.insert_payment_external_info()` to store the attempted_amount_msat in the payment_external_info sync table.

@dangeross why do we need to sync the payment_external_info with pending payment?

Reply:
Maybe I misunderstood the PR, but it looks like the intention is to store the pending amount in `attempted_amount_msat`. @dleutenegger ?

Reply:
@dangeross either this or the migration was added by mistake because I don't see any update to the sync table.
Let's wait for @dleutenegger answer.

Reply:
@roeierez yes the intention is to persist the `attempted_amount_msat` as @dangeross pointed out [here](https://github.com/breez/breez-sdk/pull/623#discussion_r1396354458), the actual call to persist just got missed in the `persist_new_bolt11_payment_attempt`-method

Reply:
Addressed in [49e5938](https://github.com/breez/breez-sdk/pull/623/commits/49e5938166a2793814d74b33000e387e93a43e9a)